apps/galaxy/src/modules/auth/SignInForm/index.tsx

```diff
@@ -0,0 +1,30 @@
+import SignInForm from "./SignInForm";
+import { Flex, Image, Text } from "@cc/ui-chakra";
+
+const SignIn = () => {
+  return (
+    <Flex height="calc(100vh - 30px)" justifyContent="center" alignItems="center">
```

Comment:
Reading this [Notes on Stack vs Flex](https://chakra-ui.com/docs/components/stack#notes-on-stack-vs-flex), looks like elements are centred by default in `@chakra-ui/react`

Reply:
![image](https://user-images.githubusercontent.com/49120762/195315557-68f5201e-e5df-4595-9229-de92e363bf00.png)

seems like it is not a default centre anymore